content/web-api-rest/studio/_index.md

```diff
@@ -0,0 +1,359 @@
+---
+title: "Studio: Consuming the Coding Events API With Postman"
+date: 2022-12-15T09:16:07-06:00
+draft: false
+weight: 5
+originalAuthor: John Woolbright # to be set by page creator
+originalAuthorGitHub: jwoolbright23 # to be set by page creator
+reviewer: Kimberly Horan # to be set by the page reviewer
+reviewerGitHub: codinglikeagirl42 # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: 12/15/22 # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+The UI of a browser is designed to make simple `GET` requests for URLs entered into its address bar. This design works great for browsing sites, but falls short when working with APIs. Anything beyond a `GET` request is difficult to send via a browser address bar alone. Think about what is needed to create a new `CodingEvent`. This type of request contains a body. Our MVC application included a view to allow us to test inputs. Our API, however, is **headless**. It does not contain the client-side form. In order to test how it handles requests then, we need a way to interact with the API server without the browser. In this studio, we work with Postman to explore how APIs can be consumed.
+
+## Setup
+
+### Install Postman
+
+If you haven't done so already, [install Postman](postman-installation)
+
+### Fork and Clone the API Source Code
+
+With Postman installed, we're ready to fork and clone the [CodingEvents API](https://github.com/LaunchCodeEducation/coding-events-api/tree/1-sqlite). Although it is implemented differently from CodingEvents MVC, you will find that most of the features from the MVC application have been supported through endpoints in the API.
```

Comment:
This links to the LaunchCode repo - the repo i used was this one ->

https://github.com/jwoolbright23/coding-events-api/tree/1-sqlite

Reply:
resolved link issues here